pkg/settings/setting.go

```diff
@@ -128,7 +128,7 @@ var (
 	GKEUpstreamRefresh                  = NewSetting("gke-refresh", "300")
 	HideLocalCluster                    = NewSetting("hide-local-cluster", "false")
 	MachineProvisionImage               = NewSetting("machine-provision-image", "rancher/machine:v0.15.0-rancher98")
-	SystemFeatureChartRefreshSeconds    = NewSetting("system-feature-chart-refresh-seconds", "900")
+	SystemFeatureChartRefreshSeconds    = NewSetting("system-feature-chart-refresh-seconds", "21600") // SystemFeatureChartRefreshSeconds Number of seconds to wait between each system chart refresh
```

Comment:
Not sure if the comment is needed, it duplicates the name without adding anything.